content/authors/anna-krystalli/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Anna Krystalli
+twitter: annakrystalli
+github: annakrystalli
+link: http://annakrystalli.me/
```

Comment:
@annakrystalli maybe this should be https? I copy-pasted it from your GitHub profile.